test_conformance/CMakeLists.txt

```diff
@@ -49,9 +49,11 @@ add_subdirectory( subgroups )
 add_subdirectory( workgroups )
 add_subdirectory( pipes )
 add_subdirectory( device_timer )
-add_subdirectory( clcpp )
-add_subdirectory( spirv_new )
-add_subdirectory( spir )
+if(KHRONOS_OFFLINE_COMPILER)
+    add_subdirectory( clcpp )
+    add_subdirectory( spirv_new )
+    add_subdirectory( spir )
```

Comment:
It's not immediately obvious to me why the `spirv_new` and `spir` tests would need `KHRONOS_OFFLINE_COMPILER`.

Reply:
Agreed. Moving those out.